app/assets/stylesheets/friendships.css.scss

```diff
@@ -0,0 +1,15 @@
+.friendship {
+  border-bottom: 1px dotted lightgrey;
+}
+
+li.pure-menu-item {
+  display: inline-block;
+  margin-right: 25px;
+}
+
+.pending {
+  border-bottom: 1px dotted green;
```

Comment:
Color `green` should be written in hexadecimal form as `#008000`<br>Color literals like `green` should only be used in variable declarations; they should be referred to via variable everywhere else.